lib/makara/railtie.rb

```diff
@@ -1,8 +1,17 @@
 module Makara
   class Railtie < ::Rails::Railtie
 
-    initializer "makara.configure_rails_initialization" do |app|
-      app.middleware.use Makara::Middleware
+    config.makara = ActiveSupport::OrderedOptions.new # enable namespaced configuration in Rails environments
+
+    initializer "makara.configure" do |app|
+      Makara.configure do |config|
+        config.skip_middleware = false
+        if app.config.makara[:skip_middleware]
+          config.skip_middleware = app.config.makara[:skip_middleware] 
+        end
+      end
+
+      app.config.middleware.insert_before "::Rails::Rack::Logger", "Makara::Middleware"
```

Comment:
Why install the middleware at all if we're going to skip it? Can we omit it entirely?